app/src/main/java/chat/rocket/android/chatdetails/presentation/ChatDetailsPresenter.kt

```diff
@@ -74,8 +74,8 @@ class ChatDetailsPresenter @Inject constructor(
         navigator.toFileList(chatRoomId)
     }
 
-    fun toMembers(chatRoomId: String) {
-        navigator.toMembersList(chatRoomId)
+    fun toMembers(chatRoomId: String, isOwner: Boolean, isMod: Boolean) {
```

Comment:
`isModerator` sounds better than `isMod`. You have even created a function with that name on `ChatRoomPresenter.kt` for instance.